### ISSUE TITLE
Собрать несколько ноутбуков  с примерами классификации

### ISSUE THREAD

**Issue by Gordondalos** [tmshv/transport-clustering#2]

**Comment by Bdata0** [comment]
1. [Mercedes cars clustering](https://www.kaggle.com/daniel89/mercedes-cars-clustering)
2. [Cars k-means clustering script](https://www.kaggle.com/thebrownviking20/cars-k-means-clustering-script)

**Comment by Bdata0** [comment]
3. [Stanford Cars Dataset with Accuracy 85%](https://www.kaggle.com/jupyternotebook/accuracy-85-with-fast-ai-in-4-epoches)
4. [PKU: Cars clustering](https://www.kaggle.com/hocop1/pku-cars-clustering)
5. [Show Cars with Similar Orientations and Clustering](https://www.kaggle.com/ren4yu/show-cars-with-similar-orientations-and-clustering)

**Comment by Bdata0** [comment]
6. [Computer Vision - Image Classification Cars](https://www.kaggle.com/mrmorj/computer-vision-image-classification-cars)
7. [CNN Stanford Cars Keras AlexNet](https://www.kaggle.com/alincijov/cnn-stanford-cars-keras-alexnet)
8. [Stanford Cars Dataset with Fastai V1](https://www.kaggle.com/paultimothymooney/stanford-cars-dataset-with-fastai-v1)

**Comment by Bdata0** [comment]
9. [cars_dataset_visualization](https://www.kaggle.com/meaninglesslives/cars-dataset-visualization)
10. [Tiny models with the Cars Dataset 70% accuracy](https://www.kaggle.com/elmahy/tiny-models-with-the-cars-dataset-70-accuracy)
11. [cars_pytorch_cnn](https://www.kaggle.com/ajaykgp12/cars-pytorch-cnn)
